docs/pages/playground.vue

```diff
@@ -21,6 +21,8 @@
         { text: 'Level 2 ', link: { path: '#' } },
       ]"
     /> -->
+    <KTable :headers="tableHeaders" :rows="tableRows" caption="User Information Table" />
```

Comment:
Just a note that in the final version, `caption` and any other strings will need to be received via prop from products, such as Kolibri, because KDS doesn't contain mechanism for translations.